gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -72,6 +71,13 @@ fluid.defaults("gpii.flowManager", {
 // Mixin grades for the FlowManager
 
 fluid.defaults("gpii.flowManager.local", {
+    // The grade to handle user login requests. Distributed from the GPII config files.
+    // In all-in-local config, the handler is "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler".
+    // In untrusted config, the handler should be "gpii.lifecycleManager.untrusted.stateChangeHandler".
+    distributeOptions: {
+        record: "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler",
+        target: "{that lifecycleManager}.options.logonStateChangeHandler"
```

Comment:
Nothing appears to read this option now? It is not present in the lifecycleManager itself.